stable/oauth2-proxy/templates/secret.yaml

```diff
@@ -12,5 +12,9 @@ type: Opaque
 data:
   cookie-secret: {{ .Values.config.cookieSecret | b64enc | quote }}
   client-secret: {{ .Values.config.clientSecret | b64enc | quote }}
+{{- if .Values.isOpenshift }}
```

Comment:
we could have it default to the generated name `{{- printf "%s" (include "oauth2-proxy.fullname" .) | b64enc | quote -}}` if clientid is left blank.